plugin/service/storage/plugin_test.go

```diff
@@ -36,6 +37,188 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+func TestStoragePlugin_NormalizeStorageBucketData(t *testing.T) {
+	ctx := context.Background()
+
+	tests := []struct {
+		name     string
+		req      *plugin.NormalizeStorageBucketDataRequest
+		expected *plugin.NormalizeStorageBucketDataResponse
+		err      string
+	}{
+		{
+			name: "missing attributes",
+			req:  &plugin.NormalizeStorageBucketDataRequest{},
+			err:  "attributes.region: missing required value \"region\"",
+		},
+		{
+			name: "empty attributes",
+			req: &plugin.NormalizeStorageBucketDataRequest{
+				Attributes: &structpb.Struct{},
+			},
+			err: "attributes.region: missing required value \"region\"",
+		},
+		{
+			name: "missing required field",
+			req: &plugin.NormalizeStorageBucketDataRequest{
+				Attributes: &structpb.Struct{
+					Fields: map[string]*structpb.Value{
+						ConstAwsEndpointUrl: structpb.NewStringValue("http://127.0.0.1:9000"),
+					},
+				},
+			},
+			err: "attributes.region: missing required value \"region\"",
+		},
+		{
+			name: "valid endpoint_url returned unchanged",
```

Comment:
Along with Hugo's comment on adding the functionality lets add a test to validate when its called with just a valid region